samples/can/README.rst

```diff
@@ -0,0 +1,45 @@
+.. _can-sample:
+
+Controller Area Network
+###########
```

Comment:
Heading underlines must be at least as long as the title itself. Add some more ``#``'s